### PR TITLE
Address a few Warning messages

### PR DESCRIPTION
**Pull request by MrPetovan**
- Address https://github.com/friendica/friendica/issues/12488#issuecomment-1445193699
- Address https://github.com/friendica/friendica/issues/12488#issuecomment-1445359764
- Address https://github.com/friendica/friendica/issues/12488#issuecomment-1448297639
- Address https://github.com/friendica/friendica/issues/12488#issuecomment-1483811015

### CODE REVIEW

**Comment by annando** [comment]
Some of the warnings seem to be hints for other problems. We shouldn't suppress the warnings here, but should have a look at why this is happening.

File: src/Content/Item.php [hunk 1]
```diff
@@ -508,7 +508,7 @@ public function expandTags(array $item, bool $setPermissions = false): array
 			}
 			$item['inform'] .= 'cid:' . $contact['id'];
 
-			if (($item['gravity'] == ItemModel::GRAVITY_COMMENT) || empty($contact['cid']) || ($contact['contact-type'] != Contact::TYPE_COMMUNITY)) {
+			if ((isset($item['gravity']) && $item['gravity'] == ItemModel::GRAVITY_COMMENT) || empty($contact['cid']) || ($contact['contact-type'] != Contact::TYPE_COMMUNITY)) {
```
Comment (annando) [inline src/Content/Item.php:511]: `Gravity` should be defined here. We shouldn't ignore this warning.

File: src/Model/Item.php [hunk 2]
```diff
@@ -625,7 +625,7 @@ public static function isValid(array $item): bool
 			return false;
 		}
 
-		if ($item['verb'] == Activity::FOLLOW) {
+		if (isset($item['verb']) && $item['verb'] == Activity::FOLLOW) {
```
Comment (annando) [inline src/Model/Item.php:628]: I guess that this happens because `verb` is missing in Statuses.php.

**Review by annando — CHANGES_REQUESTED** [review]

**Comment by nupplaphil** [comment]
What about logging these warnings as "real" warnings with context instead? So we can investigate it further

**Comment by MrPetovan** [comment]
Oh I know the cause for all of these, and I fixed them accordingly. There's no further investigation needed for these.

**Comment by MrPetovan** [comment]
@annando 
> I guess that this happens because verb is missing in Statuses.php.

Indeed, same as with gravity, and I also assume the default value isn't `Activity::FOLLOW`.

(I can't seem to be able to answer on a review conversation thread, the Ajax calls returns 500 systematically.)

**Comment by MrPetovan** [comment]
Superseded by https://github.com/friendica/friendica/pull/12963

**Pull request closed without merging**